tutorial/admin/README.md

```diff
@@ -0,0 +1,36 @@
+GEOPM TUTORIAL FOR SYSTEM ADMINISTRATORS
+========================================
+This documentation presents a tutorial for system administrators who
+would like to configure their HPC systems to enable the use of GEOPM.
+There are several different use cases that are broken out into
+numbered sections below.  The tutorial starts out with the least
+restrictive configuration of GEOPM.  The other sections show ways that
+system administrators can use GEOPM to achieve specific goals for
+energy, power and performance.
+
+The directory contains scripts that can be used to configure systems
+for use with the GEOPM runtime.  The scripts associated with each
+section will be numbered accordingly.
+
+0. Enable users to do research with the GEOPM runtime
+-----------------------------------------------------
+Follow the instructions in "RUNTIME REQUIREMENTS" section of the
+top level GEOPM README.md.
+
```

Comment:
other prerequisites depending on use cases
- how to check power capping is working
- how to check frequency control is working